src/shared/components/home/login.tsx

```diff
@@ -52,6 +57,9 @@ async function handleLoginSuccess(i: Login, loginRes: LoginResponse) {
 
   if (site.state === "success") {
     UserService.Instance.myUserInfo = site.data.my_user;
+    const isoData = setIsoData(i.context);
+    isoData.site_res.oauth_providers = site.data.oauth_providers;
+    isoData.site_res.admin_oauth_providers = site.data.admin_oauth_providers;
```

Comment:
This is really strange. IsoData is only for the initial fetch, and you should just use siteRes if you need these providers afterward.